gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=8.1.1
+version=8.1.2
 iexecCommonVersion=8.2.1
 iexecCommonsPocoVersion=3.0.4
```

Comment:
Wouldn't we want to propagate latest iexec-commons-poco here ?

Reply:
Thanks for the idea. Please see [fc19f08](https://github.com/iExecBlockchainComputing/iexec-core/pull/602/commits/fc19f088388adcbf07a275e156792ac9088a657e).